content-src/components/Header/Header.js

```diff
@@ -8,7 +8,8 @@ const Header = React.createClass({
   render() {
     const props = this.props;
     const currentRoute = props.currentRoute || {};
-    return (<header className="head">
+    return (<header className="head" hidden={props.currentRoute.path === "/"}>
```

Comment:
Not sure if we want to use `props.currentRoute.path` or just `currentRoute.path` here, since we already created an alias for it (with default empty object) on Line 10 above.